index.html

```diff
@@ -131,16 +131,22 @@
 
 ## The Hidden Cost of Bundling
 
-Bundling lowers latency and gives coherence to the whole package,
+Bundling lowers latency, lowers the protocol cost by reducing the number of
+requests and gives coherence to the whole package,
```

Comment:
What does "gives coherence to the whole package" actually mean? (I know it was there already but still)